tests/rbd/rbd_utils.py

```diff
@@ -440,6 +440,23 @@ def image_info(self, pool_name, image_name):
             return 1
         return image_info
 
+    def enable_image_feature(self, pool_name, image_name, feature_name, **kwargs):
```

Comment:
We can combine enable_image_feature and disable_rbd_feature into a single method right?